azkaban-common/src/main/java/azkaban/utils/ThinArchiveUtils.java

```diff
@@ -173,4 +178,18 @@ public static void validateDependencyHash(final File f, final Dependency d)
       throw new RuntimeException(e);
     }
   }
+
+  /**
+   * Helper method to filer out null from the set containing Dependency objects.
+   * Eg: If input set: {null, obj1, obj2 }, then output set: {obj1, obj2}
+   *
+   * @param dependencies Set of Dependency objects from which null has to be filtered out.
+   * @return Set of Dependency without any null in it.
+   */
+  public static Set<Dependency> filterNullFromDeps(Set<azkaban.spi.Dependency> dependencies) {
```

Comment:
Can you please import azkaban.spi package?

Reply:
Done.